app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -1,3 +1,57 @@
+@import 'bootstrap';
```

Comment:
This file you need only when you redefine default bootstrap styles.

Move css tags for profile_page to new file (example: profile_page.scss).